x-pack/solutions/security/plugins/security_solution/public/detections/hooks/use_alert_close_info_modal.tsx

```diff
@@ -44,13 +47,48 @@ const restartSuppressionMessageComponent = (
 const continueSuppressionMessageComponent = (
   <FormattedMessage
     id="xpack.securitySolution.alert.closeInfoModal.continueSuppressionMessage"
-    defaultMessage="Duplicate events will continue to be grouped and suppressed, but new alerts won't be created for these groups. {link}."
+    defaultMessage="Some of the alerts being closed were created while a suppression window was active. If suppression remains active, duplicate events will continue to be grouped and suppressed, but new alerts won't be created for these groups. {link}."
     values={{
       link: learnMoreLink,
     }}
   />
 );
 
+const hasAlertsInSuppressionWindow = async (params: { query?: string; ids?: string[] }) => {
+  let query;
+  if (params.ids && params.ids.length > 0) {
+    query = {
+      bool: {
+        filter: [
+          {
+            ids: {
+              values: params.ids,
+            },
+          },
+        ],
+      },
+    };
+  } else if (params.query) {
+    query = JSON.parse(params.query);
+  } else {
+    throw new Error('either query or a non empty list of alert ids must be defined');
+  }
+
+  const boolFilters = get(query, 'bool.filter', []);
```

Comment:
Not sure I understand this code completely?

Are we always in the end overwrite query bool filters to `boolFilters`. So what we pushed above is not implied?

set(query, 'bool.filter', boolFilters);


Also `query.bool?.filter.push` can give you an error if filter is undefined - more safe to do `filter?.push`

As we can get query from the string we don't control types there